app/src/main/res/values-fr/strings.xml

```diff
@@ -155,7 +159,10 @@
   <string name="vibration_profile_ring">Sonnette</string>
   <string name="vibration_profile_alarm_clock">Réveil</string>
   <string name="miband_prefs_vibration">Vibration</string>
-  <string name="pref_screen_notification_profile_sms">Notification Texto</string>
+  <string name="pref_screen_notification_profile_sms">Notification SMS</string>
+  <string name="pref_screen_notification_profile_generic_chat">Clavardage (Chat)</string>
```

Comment:
Yes, of course it's from Quebec, I forgot this.
I know there's an official word in academic French, wait for it, I'm looking for it…
OK, I think it does not really exist, we should leave "Clavardage (Chat)", I think this is clear enough.